bee-ledger/src/workers/event.rs

```diff
@@ -41,17 +43,23 @@ pub struct MessageReferenced {
 }
 
 /// An event that indicates that an output was consumed.
-#[derive(Clone)]
```

Comment:
Why are we removing this ? Events need to be cloneable otherwise we can't really use them

Reply:
fixed